### PR TITLE
fix(TDI-38563) : Some UI issues for tMarketoInput

### PR DESCRIPTION
**Pull request by undx**
**What is the current behavior?** (You can also link to an open issue here)

https://jira.talendforge.org/browse/TDI-38563

**What is the new behavior?**

Fixed UI problems.

**Please check if the PR fulfills these requirements**

- [X] The commit message follows Talend standard
- [X] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [x] The code coverage on new code >75%
- [ ] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [X] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [X] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/157/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/157/)

File: components/components-marketo/src/main/resources/org/talend/components/marketo/tmarketoinput/messages.properties [hunk 1]
```diff
@@ -27,7 +27,7 @@ property.marketoColumnName.displayName=Marketo column name
 property.maxReconnAttemps.displayName=Max reconnection attemps
 property.maxReturn.displayName=Max returm
 property.oldestCreateDate.displayName=Oldest create date
```
Comment (jzhao-talend) [inline components/components-marketo/src/main/resources/org/talend/components/marketo/tmarketoinput/messages.properties]: `oldestCreateDate` and `oldestUpdateDate` show different format?

File: components/components-marketo/src/main/resources/org/talend/components/marketo/tmarketoinput/messages.properties [hunk 2]
```diff
@@ -11,9 +11,9 @@ property.attemptsIntervalTime.displayName=Attempts interval time
 property.clientAccessId.displayName=Client access ID
 property.connection.displayName=Connection
 property.endpoint.displayName=Endpoint address
-property.fieldList.displayName=Field list
-property.latestCreateDate.displayName=Latest creation date
-property.latestUpdateDate.displayName=Latest update date
+property.fieldList.displayName=Fields (Separated by commas)
+property.latestCreateDate.displayName=Latest create date
+property.latestUpdateDate.displayName=Last Updated At
```
Comment (jzhao-talend) [inline components/components-marketo/src/main/resources/org/talend/components/marketo/tmarketoinput/messages.properties]: don't understand this modification

File: components/components-marketo/src/main/java/org/talend/components/marketo/tmarketoinput/TMarketoInputProperties.java [hunk 3]
```diff
@@ -343,7 +348,11 @@ public void afterSchema() {
         excludeTypes.type.setPossibleValues((Object[]) IncludeExcludeFieldsREST.values());
         fieldList.setValue("");
         leadKeysSegmentSize.setValue(50);
-        sinceDateTime.setValue("yyyy-MM-dd HH:mm:ss Z");
+        sinceDateTime.setValue(DATETIME_PATTERN_PARAM);
+        oldestCreateDate.setValue(DATETIME_PATTERN_PARAM);
```
Comment (jzhao-talend) [inline components/components-marketo/src/main/java/org/talend/components/marketo/tmarketoinput/TMarketoInputProperties.java]: Is it possible change date property value from string to date like what we do in salesforce
https://github.com/Talend/components/blob/master/components/components-salesforce/components-salesforce-definition/src/main/java/org/talend/components/salesforce/SalesforceGetDeletedUpdatedProperties.java#L35
So that can show a date controller in component setting to let customer select a date value.

If it can, it would be better confirm with QA and our manager,  whether this is needed.
If meed to change it,you also no need do in this pull request. you can create another task to do it.

File: components/components-marketo/src/main/resources/org/talend/components/marketo/tmarketoinput/messages.properties (hunk 2 above)
Comment (undx) [inline components/components-marketo/src/main/resources/org/talend/components/marketo/tmarketoinput/messages.properties]: Changed labels to reflect the old component used labels...

File: components/components-marketo/src/main/java/org/talend/components/marketo/tmarketoinput/TMarketoInputProperties.java (hunk 3 above)
Comment (undx) [inline components/components-marketo/src/main/java/org/talend/components/marketo/tmarketoinput/TMarketoInputProperties.java]: I kept String for dates to stay like ion the oldest components.
But, yes this should be a date. We'll do that in another PR.

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/174/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/174/)
